server/models/user.js

```diff
@@ -30,6 +33,7 @@ const UserSchema = new mongoose.Schema({
   }],
   role: {
     type: Number, // This can be an enum, should we make this an enum?
```

Comment:
We this was something @paradime commented, I think it's worth making an enum, I just haven't got around to it yet in this PR.